README.md

```diff
@@ -36,6 +36,18 @@ Beta version of the ROS2 Universal Robots driver. Should be transferred to the U
 **NOTE**: If you are a ROS2 beginner and get lost consult the [references - section](#references).
 
 
+### How to Setup the UR Robot
+
+1. To enable external control of a UR robot from a remote PC, [install URCap](/ur_robot_driver/doc/install_urcap_e_series.md).
+
+2. Switch to Remote Control by clicking on the teach pendant logo on the upper right corner.
+
+3. Create a new program. In the Installation tab of the teach pendant, select the UPCaps option on the left and adjust the IP address and port number. The remote host's IP can be found using `ifconfig`.
```

Comment:
```suggestion
3. Create a new program. In the Installation tab of the teach pendant, select the URCaps option on the left and adjust the IP address and port number. The remote host's IP can be found using `ifconfig`.
```

URCaps?

Reply:
For formatting suggestions I have found it least confusing to simply do "Insert a suggestion."